Add AMP validation on export (#6794)
zeit/next.js: errors/amp-export-validation.md

# AMP Export Validation

#### Why This Error Occurred

> During export we validate AMP pages using [amphtml-validator](https://www.npmjs.com/package/amphtml-validator), when we receive an error from the validator that means an AMP page is invalid so the export is not valid.

Reviewer: ```suggestion
During export we validate AMP pages using [amphtml-validator](https://www.npmjs.com/package/amphtml-validator). Errors received from the validator will fail the export.

Validation errors mean your page is not valid AMP HTML and will not be [indexed by AMP Caches](https://www.ampproject.org/docs/fundamentals/how_cached).
```

#### Possible Ways to Fix It

> Look at the error messages telling you the reasons for the error and update it to make the AMP page that caused the error to validate. 

Reviewer: ```suggestion
Look at the error messages and follow their appropriate links to learn more about the error and how to correct the problem.
```

# AMP Export Validation

#### Why This Error Occurred

During export we validate AMP pages using [amphtml-validator](https://www.npmjs.com/package/amphtml-validator). Errors received from the validator will fail the export.

> Validation errors mean your page is not valid AMP HTML and will not be [indexed by AMP Caches](https://www.ampproject.org/docs/fundamentals/how_cached).

Reviewer: ```suggestion
Validation errors will cause pages to not be [indexed by AMP Caches](https://www.ampproject.org/docs/fundamentals/how_cached).
```

this is less aggressive
